### DISCUSSION TITLE
Community Meeting 2022-03-23

### DISCUSSION THREAD

**Discussion by JamieMagee** [microsoft/component-detection#70] [Announcements]
📆 Wednesday 23rd March 2022
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220323T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda
- Review of current status of SPDX and CycloneDX output
	- Microsoft internal SBOM team will move discussion for adding SPDX output to component-detection repository
	- CycloneDX support is currently in prototype by @JamieMagee 

## Attendees

- @cobya (Microsoft)
- @8gitbrix (Microsoft)
- @grvillic (Microsoft)
- @annaowens (Microsoft)
- @lamac-msft (Microsoft)
- @seabass-labrax (SPDX)

## Recording

https://user-images.githubusercontent.com/1358764/159954476-52d33ae6-e037-45fd-bb1a-3695b0b47e52.mp4

**Comment by cobya** [comment]
Updated the discussion as the leader of today's meeting. Current follow ups:
- Status update from @JamieMagee on CycloneDX proof of concept
- @grvillic reaching out to Microsoft internal SBOM team for SPDX support
